newsfragments/1451.misc.rst

```diff
@@ -0,0 +1 @@
+Upgrade py-evm to 0.2.0-beta.2
```

Comment:
This is the eth-tester version. (in this eth-tester, py-evm is upgraded to https://py-evm.readthedocs.io/en/latest/release_notes.html#v0-3-0-alpha-1 )

Maybe worth noting that that version of py-evm is now 3 months old, with ~5 releases since, so another upgrade and bump of py-evm inside eth-tester might be worthwhile.

Reply:
> This is the eth-tester version

Oops, nice catch!

And upgrading eth-tester again seems like a good idea!